www/js/lib/filecache.js

```diff
@@ -0,0 +1,187 @@
+﻿/**
+ * filecache.js: Generic least-recently-used-cache used for reading file chunks.
```

Comment:
Maybe say here: "used for fast Directory Entry lookup" (instead of "used for reading file chunks").

Reply:
Or better: "Generic cache for small, frequently read file slices. It discards cached blocks according to a least-recently-used algorithm. It is used primarily for fast Directory Entry lookup, speeding up binary search."